### PR TITLE
Feature/4025 delete filter module dashboard

### PR DESCRIPTION
**Pull request by Mayons**
This PR includes the remove filter from dashboard from any module.

### CODE REVIEW

File: tests/frontend/test_kibana_app/cypress/integration/pageobjects/filters/filters.page.js [hunk 1]
```diff
@@ -7,4 +7,5 @@ export const stablishedFilter = '[data-test-subj="filter filter-enabled filter-k
 export const pinFilterAction = '[data-test-subj="pinFilter"]';
 export const selectedOperator = '[title="is"]';
 export const pinnedFilter= '[data-test-subj="filter filter-enabled filter-key-rule.level filter-value-7 filter-pinned"]';
-export const eventsButton = '//*[contains(@class,"euiTabs")]//*[contains(text(),"Events")]'
\ No newline at end of file
+export const eventsButton = '//*[contains(@class,"euiTabs")]//*[contains(text(),"Events")]'
```
Comment (mauceballos) [inline tests/frontend/test_kibana_app/cypress/integration/pageobjects/filters/filters.page.js:10]: please, End the line with ";"

File: tests/frontend/test_kibana_app/cypress/integration/pageobjects/filters/filters.page.js [hunk 2]
```diff
@@ -7,4 +7,5 @@ export const stablishedFilter = '[data-test-subj="filter filter-enabled filter-k
...
+export const removeFilterButton = '//*[contains(@class,"euiContextMenuPanel")]//*[contains(text(),"Delete")]'
```
Comment (mauceballos) [inline tests/frontend/test_kibana_app/cypress/integration/pageobjects/filters/filters.page.js:11]: please, End the line with ";"

File: tests/frontend/test_kibana_app/cypress/integration/pageobjects/filters/filters.page.js (hunk 1 above)
Comment (Mayons) [reply tests/frontend/test_kibana_app/cypress/integration/pageobjects/filters/filters.page.js:10]: Done

File: tests/frontend/test_kibana_app/cypress/integration/pageobjects/filters/filters.page.js (hunk 2 above)
Comment (Mayons) [reply tests/frontend/test_kibana_app/cypress/integration/pageobjects/filters/filters.page.js:11]: Done

**Review by mauceballos — APPROVED** [review]

File: tests/frontend/test_kibana_app/cypress/integration/features/filters/remove-filter-from-module-dashboard.feature [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      | Integrity Monitoring  |
+      | System Auditing       |
+      | Vulnerabilities       |
+      | Mitre & Attack        |
+      | GDPR                  |
+      | HIPAA                 |
+      | NIST                  |
+      | TSC                   |
+      | Policy Monitoring     |
+      | PCIDSS                |
```
Comment (MauGaP) [inline tests/frontend/test_kibana_app/cypress/integration/features/filters/remove-filter-from-module-dashboard.feature:26]: minor: add a new line to the end of file.

File: tests/frontend/test_kibana_app/cypress/integration/features/filters/remove-filter-from-module-dashboard.feature [hunk 4]
```diff
@@ -0,0 +1,26 @@
+Feature: Validate that the added filter label is remove after click remove filter option
+   
+    As Wazuh user 
+    I want to set a new filter 
+    in order to manage them
+  
+   Scenario Outline: The user remove an added filter - Module - Dashboard
```
Comment (MauGaP) [inline tests/frontend/test_kibana_app/cypress/integration/features/filters/remove-filter-from-module-dashboard.feature:7]: minor: the user 'removed and added' a filter
or the user 'removes and adds' a filter

also, the part where we say '- module - dashboard' does this mean the localization of this functionality? can it be more clearly described?

File: tests/frontend/test_kibana_app/cypress/integration/pageobjects/filters/filters.page.js [hunk 5]
```diff
@@ -7,4 +7,5 @@ export const stablishedFilter = '[data-test-subj="filter filter-enabled filter-k
 export const pinFilterAction = '[data-test-subj="pinFilter"]';
 export const selectedOperator = '[title="is"]';
 export const pinnedFilter= '[data-test-subj="filter filter-enabled filter-key-rule.level filter-value-7 filter-pinned"]';
-export const eventsButton = '//*[contains(@class,"euiTabs")]//*[contains(text(),"Events")]'
\ No newline at end of file
+export const eventsButton = '//*[contains(@class,"euiTabs")]//*[contains(text(),"Events")]';
+export const removeFilterButton = '//*[contains(@class,"euiContextMenuPanel")]//*[contains(text(),"Delete")]';
```
Comment (MauGaP) [inline tests/frontend/test_kibana_app/cypress/integration/pageobjects/filters/filters.page.js:11]: minor: add line to the end of file

File: tests/frontend/test_kibana_app/cypress/integration/pageobjects/filters/filters.page.js [hunk 6]
```diff
@@ -7,4 +7,5 @@ export const stablishedFilter = '[data-test-subj="filter filter-enabled filter-k
 export const pinFilterAction = '[data-test-subj="pinFilter"]';
 export const selectedOperator = '[title="is"]';
 export const pinnedFilter= '[data-test-subj="filter filter-enabled filter-key-rule.level filter-value-7 filter-pinned"]';
-export const eventsButton = '//*[contains(@class,"euiTabs")]//*[contains(text(),"Events")]'
\ No newline at end of file
+export const eventsButton = '//*[contains(@class,"euiTabs")]//*[contains(text(),"Events")]';
```
Comment (MauGaP) [inline tests/frontend/test_kibana_app/cypress/integration/pageobjects/filters/filters.page.js:10]: You know I don't like to use XPath selectors, so take this recommendation with a grain of salt.

In this case we could find several selectors that would work and would be easier to read and to understand, and also perform better (in execution time) but, if this is the only way you could make it work, leave it as it is and we could find a better selector in the future.

**Review by MauGaP — COMMENTED** [review]
These comments are just recommendations, we could merge this PR and fix them in a later PR if you have other open issues.

**Pull request merged**